lib/msal-react/docs/performance.md

```diff
@@ -28,19 +28,19 @@ import { NavigationClient } from "@azure/msal-browser";
  * If you would like to overwrite both you can implement INavigationClient directly instead
  */
 class CustomNavigationClient extends NavigationClient{
-    constructor(history) {
+    constructor(navigate) {
```

Comment:
Could you specify somewhere that this code snippet represents react-router v6 usage and perhaps link to the old commit for the v5 implementation?

Reply:
Updated.